plugins/gssapi.c

```diff
@@ -1210,6 +1235,29 @@ gssapi_server_mech_authneg(context_t *text,
 	    server_creds = text->server_creds;
 	}
     }
+#ifdef GSS_USE_LOAD_FROM
+    else if (accept_any && keytab != NULL) {
```

Comment:
I believe you should do this only if server_creds == GSS_C_NO_CREDENTIAL like the above code, and may make sense to rework the code to just sahre the above code path.
the only difference is text->server_name vs GSS_C_NO_NAME, just use a supporting variable and collapse the code into one code path ?

Reply:
Re-worked the code in question - not sure if it's an improvement, but does remove the near-duplicate code.